spec/queries/get_linked_spec.rb

```diff
@@ -0,0 +1,99 @@
+require "rails_helper"
+
+RSpec.describe Queries::GetLinked do
+  content_id = SecureRandom.uuid
+  target_content_id = SecureRandom.uuid
```

Comment:
Please could `#let` these. It helps to mitigate the risk of tests interfering with each other.